src/nvim/statusline.c

```diff
@@ -1507,9 +1507,6 @@ int build_stl_str_hl(win_T *wp, char *out, size_t outlen, char *fmt, char *opt_n
         str = buf_tmp;
       }
       break;
-    case STL_PAGENUM:
```

Comment:
now using `%N` in a statusline becames a hard error. is this intentional? The alternative (what vim `-printer` does) is to change this to `num = 0`

Reply:
I didn't put much thought into it. I guess it would be safer to just return 0, assuming anyone even uses this.